hugo/assets/scss/components/article.scss

```diff
@@ -69,38 +90,45 @@
         max-width: $w-content;
         padding-top: 0;
 
+        #{ $self }__info {
+            justify-content: space-between;
```

Comment:
in design the authors + meta is on the left and the tags on the right. 
Right now the meta is alligned in the center instead of the right on small screens for docs. 
![Screenshot from 2023-08-22 16-46-31](https://github.com/cue-lang/cuelang.org/assets/122357989/030d47a2-2664-40c4-95ac-2acdceaba4e8)
On blogs it does look like design (on left)